state.json

```diff
@@ -71,6 +71,15 @@
         "commands": [
           "https://status.mozilla.org/"
         ]
+      },
+      {
+        "name": "tor",
+        "commands": [
+          "http://yyz-conf-free.paas.allizom.org/",
+          "http://yyz-conf-free.paas.allizom.org/list",
+          "http://departures.moo.mx/index.html#mozilla-toronto",
+          "http://forecast.io/#/f/43.6474,-79.3942"
```

Comment:
We have a hacky way of making this present a little better on the TVs. I've modified the MV specific line to work for Toronto here:

`"content type=html content=\"<body style='height:100%;background:linear-gradient(#EAEFF2,#D4DDE4)'><iframe style='border:0;position:absolute;top:50%;left:50%;height:50%;width:30vw;height:30vh;transform:translate(-50%,-50%) scale(3);' src='http://forecast.io/embed/#lat=43.6474&lon=-79.3942&name=Mozilla Toronto&units=ca&color=#E66000'>\""`